README.md

```diff
@@ -1,62 +1,105 @@
 # samples-typescript
 
-## How to use
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-Make sure the Temporal Server is running locally. Follow the [Quick install guide](https://docs.temporal.io/docs/server/quick-install) to do that.
+- [Samples](#samples)
+  - [Basic](#basic)
+  - [API demos](#api-demos)
+    - [Activity APIs and Design Patterns](#activity-apis-and-design-patterns)
+    - [Workflow APIs](#workflow-apis)
+    - [Production APIs](#production-apis)
+    - [Advanced APIs](#advanced-apis)
+  - [Apps](#apps)
+- [Contributing](#contributing)
+  - [Config files](#config-files)
 
-Each directory contains a sample app that has a separate `README.md` with instructions on how to run that app.
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-### Contributing
+## Samples
 
-```bash
-npm install # at project root
-```
+Each directory in this repo is a sample Temporal project built with the [TypeScript SDK](https://docs.temporal.io/docs/typescript/introduction/).
 
-This repo uses `husky` and `eslint` to ensure code standards - make sure to run `npm install` at the project root, as `husky` needs to lint every git commit.
+To start a project based on one of these samples, run:
 
-External contributions are very welcome, just please make sure you agree with us on the overall direction before sending in your PR.
+```sh
+npx @temporalio/create@latest my-project-name
+```
 
-## Samples directory
+### Basic
 
-- [**Basic hello world**](https://github.com/temporalio/samples-typescript/tree/main/hello-world): Simple example of a Workflow Definition and an Activity Definition.
-  - Variant: [Basic hello world with mTLS](https://github.com/temporalio/samples-typescript/tree/main/hello-world-mtls) shows how to connect to your Temporal Cloud namespace with mTLS authentication. [Read more on the mTLS docs](https://docs.temporal.io/docs/typescript/tls).
-- [**Pure ES Modules**](https://github.com/temporalio/samples-typescript/tree/main/fetch-esm) - Example of how to configure Temporal with TyepScript and Pure ESM.
+- [**Basic hello world**](./hello-world): Simple example of a Workflow Definition and an Activity Definition.
+  - Variant: [Basic hello world with mTLS](./hello-world-mtls) shows how to connect to your Temporal Cloud namespace with mTLS authentication.
+- [**Pure ES Modules**](./fetch-esm): Configure Temporal with TypeScript and Pure ESM.
```

Comment:
just fwiw, giving the full link was somewhat intentional because its nice to click on it from the local cloned readme to get to the webpage if needed. not gonna be a stickler about it

Reply:
ah, okay—fixed back